definitions/Applications_CustomQuery.sqlx

```diff
@@ -27,7 +27,8 @@ SELECT
     reviewed_at,
     review_passed,
     age_range_min,
-    age_range_max ),
+    age_range_max,
+    subject_limited ),
```

Comment:
Interesting, I wasn't expecting this to be used in a query. I think we should consider whether we want to get the value from the application form or the region. 

- `region.subject_limited` represents whether new/current applications will be subject limited.
- `application_form.subject_limited` represents whether the application itself was subject limited, this can be different to `region.subject_limited` if a country used to be subject limited and no longer is.

Reply:
From what I can see, the values from the fields are all the same currently but I expect that is subject to change in future. I'm not sure to what extent the field is used currently or at all for current reporting, but I can rename them to `subject_limited_application` and `subject_limited_region` respectively for example and see if anything breaks/changes over time

Reply:
@willsandforddfe @thomasleese Unaware of subject_limited being used in a dashboard and i agree with the approach above - rename them to subject_limited_application and subject_limited_region respectively

Reply:
Perhaps this approach is fine then, we can ignore the value until it's needed in any dashboards.